components/grid/grid.css

```diff
@@ -0,0 +1,53 @@
+@import url('/utils.css');
+.grid-container{
+    display: grid;
+    padding: 1rem;
+    border: 1px solid black;
+    grid-template-columns: repeat(2,1fr);
```

Comment:
you can extract this and put it in box-1, like you did for box-2.

Reply:
made the changes

Reply:
the different color for grid was just to differentiate